webknossos/webknossos/annotation/annotation.py

```diff
@@ -124,6 +146,63 @@ def download(cls, annotation_path: str) -> "Annotation":
 
         return download_annotation(annotation_type, annotation_id)
 
+    def _skeleton_zip_path(self) -> Union[Openable, PathLike, str]:
+        nml_files = [i for i in self._filelist if i.endswith(".nml")]
+        assert len(nml_files) == 1
+        return _ZipPath(self._zipfile, nml_files[0])
+
+    @contextmanager
+    def _open_nml(self) -> Iterator[IO[bytes]]:
+        """
+        This method can be used to open the annotation's inner NML
+        file directly.
+        """
```

Comment:
Wouldn't this simply be the same as
```
with annotation._nml_file.open(mode="rb") as f:
    yield f
```
if we simplify the return type of `_nml_file`? I'm not sure if we need this method then, but also ok I guess.